public/css/font.css

```diff
@@ -3,206 +3,206 @@
   font-family: 'Source Code Pro';
   font-style: normal;
   font-weight: 300;
-  src: local('Source Code Pro Light'), local('SourceCodePro-Light'), url('../fonts/SourceCodePro-Light.woff') format('woff');
+  src: url('../fonts/SourceCodePro-Light.woff') format('woff');
```

Comment:
Maybe we also want to switch to woff2 fonts?

Reply:
PRs welcome, but let's focus this PR on the bug fix ;)